src/ability/ability.factory/ability.factory.ts

```diff
@@ -52,6 +52,7 @@ export class AbilityFactory {
                     in: user.voyageTeams.map((vt) => vt.memberId),
                 },
             });
+            can([Action.Manage], "Sprint");
```

Comment:
I think we should create another Action, as stated below - voyages shouldn't be able to manage sprint